CPUMeter.c

```diff
@@ -69,9 +69,12 @@ static void CPUMeter_updateValues(Meter* this) {
    double percent = Platform_setCPUValues(this, cpu);
    if (isnan(percent)) {
       xSnprintf(this->txtBuffer, sizeof(this->txtBuffer), "offline");
+      this->single_value = 0;
```

Comment:
Maybe better to omit this sanitisation? Not sure about it. Platforms that intentionally return `NAN` also set `curItems` to 0, so the graph should correctly show 0.

We must always update `single_value` though (if we ever do), so the alternative would be to always set `this->single_value = percent;` before the `if`.

Reply:
If unused I think NAN should be preferred.

Reply:
This way we indicate "we have a single value and it's zero", so it will be used.

The other way we put NAN in there and that indicates "we have no single value", in which case Meter will sum `curItems` entries, and that should be zero, so also producing zero.